Project/include/projectFunctions.php

```diff
@@ -0,0 +1,477 @@
+<?php
+
+function printHeader(){
+	echo "
+	<html>
+		<head>
+	";
+}
+
+function printMid(){
+	echo "
+		</head>
+		<body>
+	";
+}
+
+function printFooter(){
+	echo "
+		</body>
+	</html>
+	";
+}
+
+function insertUser($p_FirstName, $p_LastName, $p_UserEmail, $p_UserName, $p_PassWord){
```

Comment:
What's the `p_` prefix mean here? Normally a prefix like that would signify the type of data (int vs. string, etc.) but I don't think that's what's happening here.